Proxy/HTTP/FilterAuthenticated.bambda

```diff
@@ -0,0 +1,66 @@
+/**
+ * Filters authenticated 200 OK requests in Proxy HTTP history. See four config values below.
+ *
+ * @author joe-ds (https://github.com/joe-ds)
+ **/
+
+if (!requestResponse.hasResponse()) {
+    return false;
+}
+
+var configNoFilter = true;        // If set to false, won't show JS, GIF, JPG, PNG, CSS.
+var configNotInScopeOnly = true;  // If set to false, won't show out-of-scope items.
+var sessionCookieName = "";       // If given, will look for a cookie with that name.
+var sessionCookieValue = "";      // If given, will check if cookie with sessionCookieName has this value.
+
+var request = requestResponse.request();
+var response = requestResponse.response();
+var mimeType = requestResponse.mimeType();
+var path = requestResponse.request().pathWithoutQuery().toLowerCase();
+
+var inScope = requestResponse.request().isInScope();
+
+var isAuthorised = response.isStatusCodeClass(StatusCodeClass.CLASS_2XX_SUCCESS);
```

Comment:
Potentially could do an early return and take the `isAuthorised` check out of the final condition:
```java
// return false if response is not authorized
if (!response.isStatusCodeClass(StatusCodeClass.CLASS_2XX_SUCCESS)) {
	return false;
}
```